build.gradle

```diff
@@ -1,7 +1,8 @@
 buildscript {
 	 
     ext {
-        springBootVersion = '1.5.9.RELEASE'
+        springBootVersion = '1.5.15.RELEASE'
+		// springBootVersion = '2.0.5.RELEASE'
```

Comment:
Could we remove the comments about version 2.0.5 in this file?

Would be nice to have this clean - plus by the time we upgrade, we might already go to another version (e.g. we use 2.1.0 already).